docs/API.md

```diff
@@ -840,7 +840,7 @@ curl \
 HTTP/1.1 200 OK
 Content-Type: application/json
 
-{}
+{"tokenId":"522c251a1623e1f1db1f4fe68b9594d26772d6e77e04cb68e110c58600f97a77"}
```

Comment:
There is also a [DB_API.md](https://github.com/mozilla/fxa-auth-db-mysql/blob/c3d8e6e88e97d961ebcb038e458ef4ea5deaf9ec/docs/DB_API.md#deletesessiontokentokenid) doc that states what each database call returns. Might be helpful to add a note about sessionId.